src/server/server.js

```diff
@@ -169,7 +169,7 @@ Server.prototype._setupSentry = function (configuration) {
     _.extend(sentryOptions, configuration.sentry)
   }
 
-  this.sentry = new Sentry()
+  this.sentry = new Sentry(this.id)
```

Comment:
Is there already a consitructor for Sentry which accepts id as passed param or that is changed in a different commit?